src/api.v2/model/experiment.js

```diff
@@ -93,6 +126,7 @@ const updateSamplePosition = async (experimentId, oldPosition, newPosition) => {
 };
 
 module.exports = {
+  getAllExperiments,
   getExperimentData,
```

Comment:
I see that for `getExperimentData` and `getAllExperiments` we don't use transactions. Is this because these are not writing data to the database and there is no need to have them atomic?